oasislmf/cmd/api.py

```diff
@@ -0,0 +1,362 @@
+
+import getpass
+import io
+import json
+import sys
+
+from argparse import RawDescriptionHelpFormatter
+from pathlib2 import Path
+
+from ..utils.exceptions import OasisException
+
+from .cleaners import as_path, PathCleaner
+from .base import OasisBaseCommand, InputValues
+
+from ..api_client.client_manager import APIClient
+
+
+def load_credentials(login_arg, logger=None):
+    """
+    Load credentials from JSON file
+
+    Options:
+        1. '--api-server-login ./APIcredentials.json'
+
+        2. Load credentials from default config file
+        '-C oasislmf.json'
+
+        3. Prompt for username / password
+    """
+    if isinstance(login_arg, (str, unicode)):
+        with io.open(login_arg, encoding='utf-8') as f:
+            return json.load(f)
+
+    elif isinstance(login_arg, (dict)):
+        if set(['password','username']) <= set(login_arg.keys()):
```

Comment:
On the left hand side of line 35 instead of 

    set(['password','username'])

it is more Pythonic to use

    {'password','username'}

Because the set elements are literals which you know beforehand.